src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -22,76 +24,73 @@
      (let [attributes (:library-attributes props)
            search-result-columns (:search-result-columns props)
            extra-columns (subvec search-result-columns 4)]
-       [table/Table
-        {:ref "table" :state-key "library-table" :v 3
-         :header-row-style {:fontWeight 500 :fontSize "90%"
-                            :backgroundColor nil
-                            :color "black"
-                            :borderBottom (str "2px solid " (:border-light style/colors))}
-         :header-style {:padding "0.5em 0 0.5em 1em"}
-         :resizable-columns? true
-         :sortable-columns? true
-         :filterable? false
-         :resize-tab-color (:border-light style/colors)
-         :reorder-anchor :right
-         :reorder-style {:width "300px" :whiteSpace "nowrap" :overflow "hidden" :textOverflow "ellipsis"}
-         :reorder-prefix "Columns"
+       [Table
```

Comment:
Hmm... seems like the blocker isn't showing up.  Will investigate.  I'll also see about getting it only on the table body.